signer/src/network/libp2p/event_loop.rs

```diff
@@ -283,31 +296,43 @@ fn handle_gossipsub_event(
                 return;
             }
 
-            tracing::trace!(local_peer_id = %swarm.local_peer_id(), %peer_id,
-                "Got message: '{}' with id: {id} from peer: {peer_id}",
-                hex::encode(&message.data),
-            );
+            if tracing::enabled!(tracing::Level::TRACE) {
+                tracing::trace!(
+                    local_peer_id = %swarm.local_peer_id(),
+                    %peer_id,
+                    msg_id = hex::encode(id.0),
+                    msg = hex::encode(&message.data),
```

Comment:
Do we anticipate the entire message hex being useful? I know it's a trace log so maybe sometimes it could be?

Reply:
Yeah i wasn't sure tbh but you never know... maybe the message id (hash of the msg) is enough?